### ISSUE TITLE
refactor: repo scripts section

### ISSUE THREAD

**Issue by 0xneves** [blockful-io/swaplace-contracts#197]
### Describe the Feature Request

We want to make it easy for the command line to deploy the contracts in the repo to facilitate activities. This is part of a usability upgrade when running the `swaplace-subgraph` locally.

- [ ] Change the `deploy` script name into `deploy:swaplace` 
- [ ] Add another script as `deploy:mocks` with the deployMoks.ts script
- [ ] Remove the testnet script as it is ambiguous when simply using `yarn test --network $1`

### Describe Preferred Solution

```
/* Remove */
 "testnet": "npx hardhat test --network $1",

/* Refactor */
 "deploy": "npx hardhat run scripts/deploy.ts --network $1",

 ---

 "deploy:swaplace": "npx hardhat run scripts/deploy.ts --network $1",
 "deploy:mocks": "npx hardhat run scripts/deployMocks.ts --network $1",
```

### Related Code

[package.json](https://github.com/blockful-io/swaplace-contracts/blob/main/package.json)

**Comment by blackbeard002** [comment]
Hey @0xneves made a PR that closes this #199

**Comment by 0xneves** [comment]
> Hey @0xneves made a PR that closes this #199

I asked for a small refactor, take a look ;D
